### PR TITLE
fix: reduce react query cache time

### PR DESCRIPTION
**Pull request by mpodlasin**
Related to https://github.com/stoplightio/platform-internal/issues/7001

### CODE REVIEW

File: packages/elements-core/src/hoc/withQueryClientProvider.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { getDisplayName } from './utils';
 const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
-      staleTime: 1000 * 60 * 5, // 5 minutes
```
Comment (mallachari) [inline packages/elements-core/src/hoc/withQueryClientProvider.tsx:9]: That is 15 ms, is it intended?

Comment (mpodlasin) [reply packages/elements-core/src/hoc/withQueryClientProvider.tsx:9]: Ah damn it, you are right 😂

File: packages/elements-core/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-core",
-  "version": "7.1.7",
```
Comment (mallachari) [inline packages/elements-core/package.json:3]: Might be worth bumping dev-portal too as it's relevant to ninja, not studio only.

Comment (mpodlasin) [reply packages/elements-core/package.json:3]: God damnit, you are right again 😂

**Review by mallachari — APPROVED** [review]

**Pull request merged**